compiler/rustc_driver/src/lib.rs

```diff
@@ -1116,7 +1131,11 @@ fn extra_compiler_flags() -> Option<(Vec<String>, bool)> {
         }
     }
 
-    if !result.is_empty() { Some((result, excluded_cargo_defaults)) } else { None }
+    if !result.is_empty() {
```

Comment:
Are you using `./x.py fmt`? I'm not expecting these formatting changes.

Reply:
Ah my bad, I'm using `rustfmt` on file save with CLion, I will fix that!